Assets/Scripts/Structres/StructureCost.cs

```diff
@@ -0,0 +1,17 @@
+using Examen.Items;
+using System;
+using UnityEngine;
+
+
+namespace Examen.Structures
+{
+    [Serializable]
+    public struct StructureCost
+    {
+        [SerializeField] private Item Item;
+
+        public int Amount;
+        public readonly string ItemName => Item.Name;
+
+    }
```

Comment:
```suggestion
        public readonly string ItemName => Item.Name;
    }
```
another slight nitpick